README.md

```diff
@@ -1,16 +1,130 @@
-# BabsECS
+# Babs-ECS
 [![Build Status](https://travis-ci.org/babsnbabs/babs-ecs.svg?branch=master)](https://travis-ci.org/babsnbabs/babs-ecs)
 
-Entity Component System implemented in C++, ideally with decent error handling and acceptable performance.
+A fast, simple, cross platform Entity Component System written for C++14.
 
-## Build and Run
-1. mkdir build
-2. cd build
-3. cmake ..
-4. make
-5. ./babsecs
+There are quite a few Entity Component Systems available in C++:
+* [EntityPlus](https://github.com/Yelnats321/EntityPlus)
+* [EntityX](https://github.com/alecthomas/entityx)
+* [anax](https://github.com/miguelmartin75/anax)
+* [Artemis](https://github.com/vinova/Artemis-Cpp)
+
+This library will seem very familiar at its core and usage. It exists because we wanted a specific set of capabilities as well as the speed, but wanted a manageable codebase to work with in the event that we want to improve it. Some design goals included the following:
+* simple, small codebase
+* cross platform (linux, windows, and osx)
+* well documented and tested
+* fastest in the west
```

Comment:
Reasonable errors? I don't think we have accomplished that yet unless the custom exception counts.